wizard/WizardModeSelection.qml

```diff
@@ -121,8 +121,8 @@ Rectangle {
             }
 
             WizardMenuItem {
-                headerText: qsTr("Advanced mode") + translationManager.emptyString
-                bodyText: qsTr("Includes extra features like mining and message verification. The blockchain is downloaded to your computer.") + translationManager.emptyString
+                headerText: qsTr("Advanced mode (maximum privacy)") + translationManager.emptyString
+                bodyText: qsTr("Includes extra features like mining and message verification. By default, the blockchain is downloaded to your computer.") + translationManager.emptyString
```

Comment:
See my comment above

Reply:
If this is closed, it is related to #2236 which could also be closed.